package.json

```diff
@@ -33,10 +29,11 @@
     "eslint-plugin-prettier": "^3.1.2",
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^4.0.1",
-    "grunt-bunyan": "0.5.0",
     "mocha": "^7.1.0",
+    "onesky": "^0.1.6",
```

Comment:
Ah, this is technically different semver from previously. But given [the package is deprecated](https://github.com/overleaf/issues/issues/3329) anyway, I don't think it matters much.